drivers/adc/ad9250/ad9250.c

```diff
@@ -808,13 +808,13 @@ int32_t ad9250_jesd204b_setup(struct ad9250_dev *dev)
 		return ret;
```

Comment:
nitpick: maybe the link to the datasheet would be useful in the commit description.

Reply:
done. thanks.